corehq/apps/reports/static/reports/js/filters/case_list_explorer.js

```diff
@@ -1,24 +1,24 @@
-var suggestedCaseTypes = function(){
-    // Adds the required properties to filter the case type autocomplete dropdowns
-    var self = this;
-    self.currentCaseType = ko.observable('');
-    $('#report_filter_case_type').on('change', function(e){
-        self.currentCaseType(e.val);
-    });
-
-    self.suggestedProperties = ko.computed(function(){
-        if (self.currentCaseType() === ''){
-            return self.allCaseProperties;
-        }
-        return _.filter(self.allCaseProperties, function(prop){
-            return prop['case_type'] === self.currentCaseType() || prop['case_type'] === null;
-        });
-    });
-};
-
 hqDefine("reports/js/filters/case_list_explorer", ['jQuery', 'underscore', 'knockout'], function($, _, ko) {
     'use strict';
 
+    var suggestions = function(){
+        // Adds the required properties to filter the case type autocomplete dropdowns
+        var self = this;
+        self.currentCaseType = ko.observable('');
+        $('#report_filter_case_type').on('change', function(e){
```

Comment:
This looks like you're setting up basic binding between an HTML element and a js value - can you use knockout's `val` here?

Reply:
The case type filter shown on this page isn't a knockout filter. I figured this was the easiest way of getting the values of these other filters to update when that filter changes. Do you think there's a better way short of rewriting the case type filter?

Reply:
Nope, I misunderstood (and I made this comment before reading more and realizing that you know what you're doing with knockout).